libs/model/src/aggregates/community/CreateCommunity.command.ts

```diff
@@ -45,7 +46,7 @@ function baseToNetwork(n: ChainBase): ChainNetwork {
 export function CreateCommunity(): Command<typeof schemas.CreateCommunity> {
   return {
     ...schemas.CreateCommunity,
-    auth: [],
+    auth: [authVerified()],
```

Comment:
It should just be required on all secure routes unless auth contains specific alternatives.

Reply:
added to all commands that are user facing... missing anything?

Reply:
I meant for it to be enabled by default as part of the framework not in that you have to explicitly add it to all the commands. Is there ever a case where the address shouldn't be verified when a JWT + address header is provided?